### ISSUE TITLE
bug build fails yabause-runner no rqueriments check/info provided

### ISSUE THREAD

**Issue by mckaygerhard** [Yabause/yabause#71]
no detection for requerimients of the yabause-runner tests , no info for build (requeriments)
also no detection of the requerimients of the featured screenshot recent commit merge: #65

so then possible ways:
1) autodetect what minimun version are necesary to build that feature, if not found disableit (i not revise so deep but i dont see any cahnges in ui ports gtk/qt/androit yet, note i have a very old androit)
2) made downgrade of necesary and use only C build event C++ and modern dependes, this made able to build for many others devices and olders but capable pc to emulate sega saturn games (please i not buy a dual core machine to use a older console with prematue poligon support, play are emulate in olders pentiums with very ancient linuxes )

the error i got are: seems are problem of the include paths/dependencs
```
Scanning dependencies of target yabause-runner
[100%] Building CXX object src/runner/CMakeFiles/yabause-runner.dir/yui.cpp.o
/media/DATASUPER/MASSENKOH-new/gamesemus/yabause/yabausegit/yabause/src/runner/yui.cpp:37:29: error: lodepng/lodepng.h: No existe el fichero o el directorio
/media/DATASUPER/MASSENKOH-new/gamesemus/yabause/yabausegit/yabause/src/runner/yui.cpp:38:31: error: lodepng/lodepng.cpp: No existe el fichero o el directorio
/media/DATASUPER/MASSENKOH-new/gamesemus/yabause/yabausegit/yabause/src/runner/yui.cpp: In function ‘bool handle_screenshot(bool, std::string, std::string, int)’:
/media/DATASUPER/MASSENKOH-new/gamesemus/yabause/yabausegit/yabause/src/runner/yui.cpp:190: error: ‘lodepng’ has not been declared
/media/DATASUPER/MASSENKOH-new/gamesemus/yabause/yabausegit/yabause/src/runner/yui.cpp:194: error: ‘lodepng_error_text’ was not declared in this scope
/media/DATASUPER/MASSENKOH-new/gamesemus/yabause/yabausegit/yabause/src/runner/yui.cpp:204: error: ‘vector’ is not a member of ‘std’
/media/DATASUPER/MASSENKOH-new/gamesemus/yabause/yabausegit/yabause/src/runner/yui.cpp:204: error: expected primary-expression before ‘unsigned’
/media/DATASUPER/MASSENKOH-new/gamesemus/yabause/yabausegit/yabause/src/runner/yui.cpp:204: error: expected ‘;’ before ‘unsigned’
/media/DATASUPER/MASSENKOH-new/gamesemus/yabause/yabausegit/yabause/src/runner/yui.cpp:205: error: ‘vector’ is not a member of ‘std’
/media/DATASUPER/MASSENKOH-new/gamesemus/yabause/yabausegit/yabause/src/runner/yui.cpp:205: error: expected primary-expression before ‘>’ token
/media/DATASUPER/MASSENKOH-new/gamesemus/yabause/yabausegit/yabause/src/runner/yui.cpp:205: error: ‘correct_image_u32’ was not declared in this scope
/media/DATASUPER/MASSENKOH-new/gamesemus/yabause/yabausegit/yabause/src/runner/yui.cpp:209: error: ‘lodepng’ has not been declared
/media/DATASUPER/MASSENKOH-new/gamesemus/yabause/yabausegit/yabause/src/runner/yui.cpp:209: error: ‘correct_image’ was not declared in this scope
/media/DATASUPER/MASSENKOH-new/gamesemus/yabause/yabausegit/yabause/src/runner/yui.cpp:213: error: ‘lodepng_error_text’ was not declared in this scope
/media/DATASUPER/MASSENKOH-new/gamesemus/yabause/yabausegit/yabause/src/runner/yui.cpp: In function ‘int main(int, char**)’:
/media/DATASUPER/MASSENKOH-new/gamesemus/yabause/yabausegit/yabause/src/runner/yui.cpp:422: warning: deprecated conversion from string constant to ‘char*’
/media/DATASUPER/MASSENKOH-new/gamesemus/yabause/yabausegit/yabause/src/runner/yui.cpp:501: warning: deprecated conversion from string constant to ‘char*’
make[2]: *** [src/runner/CMakeFiles/yabause-runner.dir/yui.cpp.o] Error 1
make[1]: *** [src/runner/CMakeFiles/yabause-runner.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by mckaygerhard** [comment]
aja! how to use it? http://lodev.org/lodepng/

**Comment by Guillaumito** [comment]
Fixed by 512599989668a160f6256fe41675271e6e4094b7

**Comment by mckaygerhard** [comment]
ok this little check are a nasty hack for now.. 

in the right way, what must i have to enable this on some ports, by example, for olders androits can be done? (i already know u have also a older androit device je je)

answer in the issue or in the commit comment .. as u prefers i think its better in the commit
